azurerm/helpers/azure/app_service.go

```diff
@@ -316,6 +316,17 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 								Optional:     true,
 								ValidateFunc: validation.StringIsNotEmpty,
 							},
+							"name": {
+								Type:         schema.TypeString,
+								Optional:     true,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+							"priority": {
+								Type:         schema.TypeInt,
+								Optional:     true,
+								Default:      65000,
+								ValidateFunc: validation.IntBetween(1, 2147483647),
+							},
```

Comment:
I think this should be `Computed: true,` rather than setting a default. If more than one IP Restriction is used this could cause a conflict.  The "default" value is assigned by the platform, rather than it being in the SDK, so can safely be read back from the API if not explicitly set.